packages/server/src/api/rest/index.ts

```diff
@@ -32,6 +32,9 @@ const urlPatterns = {
     relationship: new UrlPattern('/:type/:id/relationships/:relationship'),
 };
 
+export const idDivider = '_';
+export const compoundIdKey = 'compoundId';
```

Comment:
I'm wondering if we actually need this field to be a fixed name. Can we just use Prisma's convention (id1_id2_...) as the compound key field name?